spec/controllers/events_controller_spec.rb

```diff
@@ -21,4 +22,13 @@
       expect(response).to have_http_status(:success)
     end
   end
+
+  describe '#create' do
+    it 'Events can be created' do
+      post :create, event: FactoryGirl.attributes_for(:event)
+      event.reload
```

Comment:
`event` does not reference the event you just created, it runs the factory and makes a new one.